demo/Alsa cpp/compile.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+echo "Building Porcupine Alsa Demo.."
+
+SYSTEM=`uname`
+MACHINE="`uname -m`"
+LIB="NONE"
+
+
+if [ $SYSTEM = "Darwin" ]
+then
+    SYSTEM="mac"
+    LIB="-L../../lib/mac/$MACHINE"
+fi
+
+if [ $SYSTEM = "Windows" ]
+then
+    SYSTEM="windows"
+    LIB="-L../../lib/windows/$MACHINE"
+fi
+
+if [ $SYSTEM = "Linux" ]
+then
+    if [[ $MACHINE == arm* ]];
+    then
+        SYSTEM="raspberry-pi"
```

Comment:
This is a tricky one. Because this is the path for lib which is optimized for RPi zero. If someone wants to use RPi2/3 they need to use the other provided libs. Maybe leave this as an input argument. I am open to suggestions. Donno if there is an easy way to find the type of RPi reliably across models.